gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,6 @@
+#Tue Dec 18 10:40:21 MSK 2018
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.8-bin.zip
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.8.1-bin.zip
```

Comment:
Are there any new capabiilities in Gradle 6.8.1 for `js(BOTH)` over 6.8?
If not, it would be better to leave the current Gradle version until the next library release.

Reply:
There aren't, I'll discard this change